std/file.d

```diff
@@ -4898,6 +4902,9 @@ public:
     directory is traversed. The name of each iterated directory entry
     contains the absolute or relative _path (depending on _pathname).
 
+    $(NOTE It's not guaranteed to return a sorted iterator as it depends on the
+    filesystem implementation)
```

Comment:
```suggestion
    $(NOTE The order of returned directory entries is as it is provided by the
    operating system / filesystem, and may not follow any particular sorting.)
```

Reply:
```
/dev/shm/dtest/work/repo/dlang.org/web/phobos-prerelease/std_file.html:1544: [UNDEFINED MACRO] </div><div class="description"><!--UNDEFINED MACRO: "NOTE"-->
```

I thought NOTE macro was a thing, apparently, it doesn't.